packages/collector-main/src/getWebpackConfig.ts

```diff
@@ -0,0 +1,37 @@
+import { getMocksPath, getOutputPath } from './pathFactories';
+
+export const getWebpackConfig = ({ executionPath, entry, webpack, componentPaths }) => ({
+  entry,
+  output: {
+    path: getOutputPath(executionPath),
+    filename: '[name].js',
+    libraryTarget: 'umd',
+  },
+  module: {
+    rules: [
+      {
+        test: /\.jsx?$/,
+        loader: 'babel-loader',
+      },
+    ],
+  },
+  externals: {
+    '@bojagi/collector-main': '@bojagi/collector-main',
+    react: 'react',
+  },
+  mode: 'development',
+  plugins: [
+    ...componentPaths.map(
+      path =>
+        new webpack.NormalModuleReplacementPlugin(
+          new RegExp(path.replace(/\./g, '.').replace(/\//g, '/')),
```

Comment:
what does this do? for me it looks like its replacinv all `.` with `.` and all `/` with `/`, so basically nothing

Reply:
oh yeah, I forgot, it should replace it with `\.` and `\\`.
Will change it.

Reply:
ah, it was prettier/eslint that removed it :P